integration_tests/commands/resp/set_data_cmd_test.go

```diff
@@ -0,0 +1,164 @@
+package resp
+
+import (
+	"gotest.tools/v3/assert"
```

Comment:
we can make use of github.com/stretchr/testify/assert package here. It provides us with variety of functions to test equality of arrays, map etc.

Reply:
Yes, we can, we can do revamp the package again once we have migration complete.